static/js/urlparser.js

```diff
@@ -0,0 +1,15 @@
+function getUrlParam(key) {
```

Comment:
We normally add third party JS libraries to the `static/js/` directory, so you can place urlparser.js to the root of the repository.

Reply:
Done